git_machete/client.py

```diff
@@ -68,6 +68,10 @@ def up_branch(self) -> Dict[LocalBranchShortName, Optional[LocalBranchShortName]
     def up_branch(self, val: Dict[LocalBranchShortName, Optional[LocalBranchShortName]]) -> None:
         self._up_branch = val
 
+    def get_childless_branches(self):
+        parent_branches = self.__down_branches.keys()
```

Comment:
Also, I'm not 100% sure if it's guaranteed (in general case, esp. after some slide-outs have been performed in the given git-machete execution) that all values in `__down_branches` are non-empty lists. I'd only leave the keys which correspond to non-empty branch lists